Sources/NetworkProtection/Diagnostics/NetworkProtectionConnectionBandwidthAnalyzer.swift

```diff
@@ -111,8 +111,16 @@ final class NetworkProtectionConnectionBandwidthAnalyzer {
 
     private func bytesPerSecond(newer: Snapshot, older: Snapshot) -> (rx: Double, tx: Double) {
```

Comment:
Since there was a crash related to this, I‘d suggest converting this function into `static` (as it‘s not using the state) and adding a test for it